File: prime/prime.py
#!/usr/bin/env python3

import math

N = 100

def isprime(n):
    r = math.sqrt(n)
    for i in range(2, int(r)+1):
        if n % i == 0:
            return False
    return True

for i in range(2, N):
    if isprime(i):
        print(i, end=' ')
print()

File: prime/prime.rs
const N: i32 = 101;

fn is_prime(n: i32) -> bool {
    let r = (n as f64).sqrt() as i32;
    for i in 2..r + 1 {
        if n % i == 0 {
            return false;
        }
    }
    true
}

fn main() {
    for i in 2..N {
        if is_prime(i) {
            print!("{} ", i);
        }
    }
    println!()
}
